Frontend/src/packages/dashboard/api/index.ts

```diff
@@ -1,4 +1,4 @@
-import { toFileOrFolder } from './helpers';
+import { toFileOrFolder} from './helpers';
```

Comment:
being nit picky about code style but, spaces between square brackets should be consistent, i.e. it should be 
`{ toFileOrFolder }`